README.md

```diff
@@ -1,71 +1,157 @@
 # GitHub Command Line Tool
 
-Human-friendly command line tool for the GitHub API.
+A human-friendly\* command line tool for the GitHub API.
```

Comment:
you could alternatively write `<sup>1</sup>`<sup>1</sup>

<sup>1</sup> like this

Reply:
Thanks :)